src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -1,54 +1,138 @@
 package org.broadinstitute.consent.http.models;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.CaseFormat;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
 
 public class DataAccessRequest {
 
-    @JsonProperty
-    public Integer id;
+  private static final List<String> DEPRECATED_PROPS = Arrays.asList("referenceId", "investigator",
+      "institution", "department", "address1", "city", "zipcode", "zipCode", "state", "country",
```

Comment:
Wasn't there an address2 property as well?

Reply:
Good catch, will add that